packages/core/src/components/editable-text/editableText.tsx

```diff
@@ -105,6 +105,13 @@ export interface IEditableTextProps extends IntentProps, Props {
      */
     selectAllOnFocus?: boolean;
 
+    /**
+     * Whether the input should stop the event propagation when escape is pressed
+     *
+     * @default false
+     */
+    stopPropagation?: boolean;
```

Comment:
I think this prop name should be more explicit:

```suggestion
    stopEscapeKeyPropagation?: boolean;
```